src/context/useSmallScreen.tsx

```diff
@@ -2,12 +2,20 @@ import useEventListener from "util/useEventListener";
 import { useState } from "react";
 import { isWidthBelow } from "util/helpers";
 
-export const useSmallScreen = (): boolean => {
-  const [isSmallScreen, setIsSmallScreen] = useState(isWidthBelow(620));
+export const smallScreenSize = 620;
+export const mediumScreenSize = 820;
+export const largeScreenSize = 1200;
+
+export const useSmallScreen = (
+  currentScreenSize = smallScreenSize,
```

Comment:
Naming is a bit confusing. The parameter we pass in here is the breakpoint between small and not small, not really a current size.